src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OrganisationsController.cs

```diff
@@ -150,7 +152,36 @@ public async Task<ActionResult<CreateOrganisationResponseModel>> CreateOrganisat
 
             response.OrganisationId = result.Value;
 
-            return CreatedAtAction(nameof(GetByIdAsync).TrimAsync(), null,  new { id = result.Value }, response);
+            return CreatedAtAction(nameof(GetByIdAsync).TrimAsync(), null, new { id = result.Value }, response);
+        }
+
+        [HttpGet]
+        [Route("{id}/service-recipients")]
+        public async Task<ActionResult> GetServiceRecipientsAsync(Guid id)
```

Comment:
I'd recommend updating the return type to `Task<ActionResult<IEnumerable<ServiceRecipientsModel>>>`.

Reply:
👍